### PR TITLE
Doc on batch editing and portals.

### PR DESCRIPTION
**Pull request by fxprunayre**

### CODE REVIEW

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
+.. _portal-configuration:
+
+Portal configuration
+####################
+
+From the ``admin console`` > ``settings`` user can configure portal under the ``sources`` section.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: I'd rephrase: Portals can be configured under the sources section in admin console > settings

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+
+.. figure:: img/portal-access.png
+
+
+There are 3 types of sources:
+
+ - the main catalogue,
+
+ - sub-portal which can be a subset of the main catalogue
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: sub-portals, which can be a subset of the main catalogue

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+ - harvesters.
+
+
+.. figure:: img/portal-types.png
+
+
+The main catalogue correspond to the current installation.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: correspond -> corresponds

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+When harvesting, at least one source is created and represent the harvester. When harvesting from another GeoNetwork node,
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: at least one source is created to represent the harvester

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+sources from the target catalogue are also harvested to keep track of the true origin of records. This only applies
+to the GeoNetwork protocols which use MEF (::ref:`mef_format`) which contains the source catalogue information.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: protocols using MEF, which contains...

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+The last type is sub portal and is described in details below.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: sub-portal, which is described in the details below.

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+
+Configuring a sub portal
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: sub-portal (to be consistent)

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+````````````````````````
+
+Sub portal can be used to create a space in the main catalogue only focusing on a subset of records.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: Sub-portals can be used to create...

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+
+A sub portal is defined by:
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: sub-portal

**Comment by fxprunayre** [comment]
@archaeogeek thanks for you proposals. for English and wording changes, maybe it could be more efficient, if you directly update the file in that PR and commit - What do you think ?

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+
+.. figure:: img/portal-subportal.png
+
+
+When creating a subportal with an identifier ``ìnspire`` for example, a new entry point on your catalogue will be available:
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: sub-portal

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+http://localhost:8080/geonetwork/inspire/. Accessing the catalogue through it will only provide access to records
+matching the filter defined for this portal. The ``search filter`` parameter value use the Lucene query parser syntax
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: use -> uses

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+
+User privileges apply as in the main instance.
+
+
+The logo and name of the portal will be displayed instead of the main instance information:
+
+.. figure:: img/portal-header.png
+
+
+A CSW service is also available for this portal http://localhost:8080/geonetwork/inspire/eng/csw (and replace the virtual CSW feature).
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: and replace -> replacing

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+.. figure:: img/portal-list.png
+
+
+Example of usage
+````````````````
+
+Creating an INSPIRE directive space
+-----------------------------------
+
+For the INSPIRE directive, catalogue administrator needs to publish an entry point providing access only to INSPIRE related records. An INSPIRE subportal can be created with a filter on keywords ``+thesaurusName:"GEMET - INSPIRE themes, version 1.0"``.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: a catalogue administrator OR the catalogue administrator

Comment (archaeogeek) [reply source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: also sub-portal again

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+
+
+
+Creating a space for partners
+-----------------------------
+
+Some organizations are opening the catalogue to a set of partners. In such case, each partner access the catalogue
+and create their records in dedicated groups usually. A good example is https://www.geocat.ch/
+providing a main search filter ``catalogue``.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: Some organizations need to open the catalogue to a set of partners. In such cases, each partner generally accesses the catalogue and creates their records in dedicated groups

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+
+.. figure:: img/portal-geocatch.png
+
+The concept of sub portal adds the possibility to create dedicated URL for each partner. The header can contain
+the partner identification with name and logo and optionnaly the user interface can also be customized
+(see :ref:`user-interface-configuration`
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: The concept of a sub-portal allows the possibility of creating a dedicated URL for each partner. The header can contain the partner identification with name and logo and optionally the user interface can be customized

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+
+- One portal for each partner with a filter matching records in that group
+
+
+
+To configure this, apply the following steps:
+
+- Create a group for the partner eg. ``oca`` (see :ref:`creating-group`).
+
+- Create at least one user for the partner (see :ref:`creating-user`). User created must be member of the group ``oca``. If you want user to be able to configure the portal (eg. change the name, choose a logo), user must have at least the ``UserAdmin`` profile for the group ``oca``.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: Create at least one user for the partner (see :ref:`creating-user`). The user must be member of the group ``oca``. If you want the user to be able to configure the portal (eg. change the name, choose a logo), the user must have at least the UserAdmin profile

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+
+- Create a portal. This portal can have the same name. eg. ``oca``. The portal filter can be based on the fact that a record published in the group ``oca`` is in this portal. Such filter is ``+_groupPublished:oca``. Once created the portal is accessible at http://localhost:8080/geonetwork/oca.
+
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: Create a sub-portal. This can have the same name as the group, eg. ``oca`` but this is not essential. The filter can be created using the fact that a record published in the group ``oca`` should be in this sub-portal, using the syntax ``+_groupPublished:oca``. Once created the sub-portal

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+- (Optional) Link the portal to a user admin group to allow ``UserAdmin`` to configure their portal.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: portal -> sub-portal (x2)

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+
+
+If you make this type of configuration ie. one partner = one group = one portal and users are only member of one group, then when connecting to a partner portal:
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: With this type configuration,  ie. one partner = one group = one sub-portal and users are only member of one group, then when connecting to a partner sub-portal:

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+
+- user will only see records published in that group in the entire application
+
+- when creating new record, no group selection is provided because user is member of one group only
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: record -> records

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+
+.. figure:: img/portal-oca-newrecord.png
+
+
+
+Remember that a record is visible in the ``oca`` portal because it is published in the group ``oca``:
+
+.. figure:: img/portal-oca-privileges.png
+
+If the ``publish`` operation is removed from ``oca`` group, then record will not be visible anymore in that portal.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: then records will no longer be visible in that sub-portal

File: source/administrator-guide/configuring-the-catalog/portal-configuration.rst [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+
+
+
+
+In some situation, you also want to share templates among partners. 2 options for this:
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/portal-configuration.rst]: situation -> situations

There are 2 options for this:

File: source/administrator-guide/configuring-the-catalog/virtual-csw-configuration.rst [hunk 23]
```diff
@@ -4,6 +4,9 @@
 Configuring virtual CSW
 #######################
 
+Warning: This feature will be deprecated in future version. Use :ref:`portal_documentation` instead.
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/virtual-csw-configuration.rst]: a future version (perhaps with an idea of when)

File: source/administrator-guide/configuring-the-catalog/virtual-csw-configuration.rst [hunk 24]
```diff
@@ -4,6 +4,9 @@
...
+
+
 This feature of CSW server adds the capability to create custom CSW entry points that apply extra criteria to the CSW requests, allowing to implement several useful cases like, for example:
```
Comment (archaeogeek) [inline source/administrator-guide/configuring-the-catalog/virtual-csw-configuration.rst]: This feature adds the capability
